send/ad_group_vsup

```diff
@@ -79,7 +79,7 @@ print "Group failed to update (members): " . $counter_group_members_not_updated.
 
 $lock->unlock();
 
-if ($counter_group_members_updated_with_errors > 0) { die "Failed to process: " . $counter_group_members_updated_with_errors . " entries.\nSee log at: ~/send/logs/$service_name.log";}
+if ($counter_group_members_updated_with_errors > 0) { die "Failed to process: " . $counter_group_members_updated_with_errors . " entries.\nSee log at: ~/perun-engine/send/logs/$service_name.log";}
```

Comment:
When I think about it, logs on instance managed by ansible will be probably in a different location, so the path should be different. This apply to other scripts too.